tools/scenario-player/scenario_player/tasks/channels.py

```diff
@@ -19,7 +19,7 @@ def _request_params(self):
             ],
         )
         total_deposit = self._config.get('total_deposit')
-        if total_deposit:
+        if total_deposit or total_deposit == 0:
```

Comment:
IMO it would be cleaner to write `if total_deposit is not None`

Reply:
Yea. Will do that after I test it out. I also thought that it makes no sens the or.

Reply:
The problem here is that `0` is falsy in python